README.md

```diff
@@ -53,14 +53,14 @@ On release, automated continuous integration tests run the pipeline on a [full-s
 
 ## Quick Start
 
-1. Install [`nextflow`](https://nf-co.re/usage/installation)
+1. Install [`Nextflow`](https://nf-co.re/usage/installation) (`>=21.04.0-edge`); see [usage docs](https://nf-co.re/rnaseq/usage#nextflow-edge-releases) if the pipeline requires an `edge` release of Nextflow.
```

Comment:
I feel that the this doc could be part of the nf-core docs, and not only part of the rnaseq pipeline

Reply:
Yup. I have mentioned that already on Slack but don't really know where the most appropriate place to put them is. Hopefully, should just be a copy and paste when we decide then we can remove them from here.

Reply:
See https://github.com/nf-core/nf-co.re/pull/758